sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala

```diff
@@ -265,7 +265,7 @@ object SidechainNodeViewHolderRef {
             genesisBlock: SidechainBlock)
            (implicit system: ActorSystem): ActorRef =
     system.actorOf(props(sidechainSettings, historyStorage, consensusDataStorage, stateStorage, forgerBoxStorage, utxoMerkleTreeProvider, walletBoxStorage, secretStorage,
-      walletTransactionStorage, forgingBoxesInfoStorage, cswDataProvider, backupStorage, params, timeProvider, applicationWallet, applicationState, genesisBlock).withMailbox("akka.actor.deployment.prio-mailbox"))
```

Comment:
`.withMailbox("akka.actor.deployment.prio-mailbox"))` is critical to have because we need to prioritize some messages.
Why was it removed from this and other places?

Reply:
It was removed because in `props` method is already called

Reply:
got it, thanks!